config.go

```diff
@@ -7,9 +7,35 @@ import (
 	"time"
 )
 
+// These are the versions of the protocol (which includes wire protocol as
```

Comment:
by wire protocol, you mean the message fields sent over the network, but not their encoding?

Also, are snapshots included in this protocol version or not?